directories/src/components/storage/index.js

```diff
@@ -3,15 +3,12 @@ import { connect } from "react-redux";
 import { Input, Button, Container, Header, Image } from "semantic-ui-react";
 import _ from "lodash";
 
```

Comment:
Was it intentional to remove all those imports from this file?

Reply:
Yes, I removed unused imports in the storage/index.js (which is currently only used for the treasure finding interface, as well as the unused iota import in utils/sidechain.js.

The storage.js only dispatched 1-2 functions that did not have any functionality at that point - I assume Edmund will create a new file/directory for initializing the webnode since storage.js was repurposed for the treasure finding interface. Renaming it might make sense, but I will check on that with Edmund when he is online